tensorflow/src/main/scala/com/spotify/featran/tensorflow/package.scala

```diff
@@ -19,10 +19,12 @@ package com.spotify.featran
 
 import org.tensorflow.example.{Example, Features}
 import org.tensorflow.{example => tf}
+import _root_.java.util.regex.Pattern
 
 package object tensorflow {
+  private val featureNameNormalization = Pattern.compile("[^A-Za-z0-9_]")
```

Comment:
Patterns serializable or should this be lazy?  Although not quite clear to me at a brief glance if serialization would be attempted here.

Reply:
@richwhitjr it's inside an object so we get serialization for "free".

Reply:
Also Pattern is serializable.

Reply:
@ravwojdyla 🎨 can we make it FeatureNameNormalization?

Reply:
@regadas why?

Reply:
to follow scala style convention on constant names.

Reply:
Does this codebase follow that rule tho, see:
 * MultiFeatureExtractor
 * FlinkCollectionType
 * MDLPDiscretizer

?

It seems like the above is the most common here.

Reply:
that's why I marked the comment with 🎨 and not requested changes 😄 it's up to you

Reply:
👍